docs/user_guide/source/api/io.rst

```diff
@@ -61,7 +61,7 @@ Each Engine allows the user to fine tune execution of buffering and output tasks
 
     /** Signature */
     /** Passing several parameters at once */
-    void SetParameters(const adios2:Params& parameters);
+    void SetParameters(const adios2:Params parameters);
```

Comment:
Pointers * and references & have been removed from the returned objects, not the function arguments. They still need the distinction between passing by reference and passing by value. Please double check with the current APIs.